app/controllers/grant_reviewers/invitations_controller.rb

```diff
@@ -0,0 +1,31 @@
+module GrantReviewers
+  class InvitationsController < ApplicationController
+    before_action :set_grant
+
+    def index
+      authorize @grant, :edit?
+      @invitations = GrantReviewer::Invitation.by_grant(@grant)
+      skip_after_action :verify_policy_scope
+    end
+
+    def update;end
+
+    def create
+      authorize @grant, :edit?
+      invitation = GrantReviewer::Invitation.create(grant: @grant, inviter: current_user, email: params[:email])
+      if invitation.errors.none?
```

Comment:
there can be other type of failures such as database connection error, in addition to what .errors() returns which is specfic to activerecord validation errors.
The standard way is to do a .new() first and check if .save() returns true